packages/map-template/src/components/Map/GoogleMapsMap/GoogleMapsMap.jsx

```diff
@@ -29,6 +30,7 @@ function GoogleMapsMap({ onMapView, onPositionControl }) {
     const primaryColor = useRecoilValue(primaryColorState);
     const bearing = useRecoilValue(bearingState);
     const pitch = useRecoilValue(pitchState);
+    const isDesktop = useMediaQuery('(min-width: 992px)');
```

Comment:
We now have this `min-width: 992px` in many places, making it harder to change. Can we do anything about that?

Reply:
Maybe we could have a helper function instead which would return a boolean if it's desktop? What do you think?

Reply:
Something like that yes 😄

Reply:
Or a new custom hook that uses the useMediaQuery hook.

Reply:
Refactored the code and added a custom hook that uses the useMediaQuery hook.

Reply:
👏 🙌